### PR TITLE
TW-688 Avoid Freeze when uploading file

### PR DESCRIPTION
**Pull request by drminh2807**
# Depend on
- https://github.com/linagora/matrix-dart-sdk/pull/21

# Issue
- #688 

# Demo
Screenshot 2023-09-27 at 10 06 32

https://github.com/linagora/twake-on-matrix/assets/12546908/ce9296df-ea24-4318-a2c5-e8f38d86ce6d

https://github.com/linagora/twake-on-matrix/assets/12546908/e89d351f-1f60-491c-ae83-469cbdc48f07

# Web performance
Screenshot 2023-09-27 at 21 38 30
Screenshot 2023-09-28 at 11 36 51

# Mobile performance
signal-2023-09-27-213302_002
Screenshot 2023-09-28 at 11 22 30

### CODE REVIEW

File: lib/utils/js_window/universal_image_bitmap.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class UniversalImageBitmap {
+  int? height;
+  int? width;
+  UniversalImageBitmap({
+    this.height,
+    this.width,
+  });
+}
```
Comment (nqhhdev) [inline lib/utils/js_window/universal_image_bitmap.dart:8]:
```suggestion
class UniversalImageBitmap {
  final int? height;
  final int? width;
  UniversalImageBitmap({
    this.height,
    this.width,
  });
}
```

WDYT

Comment (drminh2807) [reply lib/utils/js_window/universal_image_bitmap.dart]: done

**Review by nqhhdev — APPROVED** [review]

**Comment by sherlockvn** [comment]
> # Issue
> * [Avoid Freeze when uploading file #688](https://github.com/linagora/twake-on-matrix/issues/688)
> 
> # Demo
> Screenshot 2023-09-27 at 10 06 32
>  screen-20230927-182053.mp4 
>  Screen.Recording.2023-09-27.at.21.50.24.mov 
> # Web performance
> Screenshot 2023-09-27 at 21 38 30
> 
> # Mobile performance
> signal-2023-09-27-213302_002

Could you demo with large image files and videos also ?

**Comment by drminh2807** [comment]
> Could you demo with large image files and videos also ?

@sherlockvn  run benchmarked , see description

**Review by sherlockvn — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**